scripts/postgres_test_setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
@kenxu95 I think using these scripts to run / teardown a postgres instance makes sense, but would let you verify